manifests/manifests.go

```diff
@@ -347,11 +347,11 @@ func (l *list) Add(ctx context.Context, sys *types.SystemContext, ref types.Imag
 				instanceInfo.Architecture = config.Architecture
 			}
 		}
+		manifestBytes, manifestType, err = src.GetManifest(ctx, instanceInfo.instanceDigest)
```

Comment:
Could this be `:=` to make it clearer that the variable only needs to exist for the duration of the single loop iteration?

Or perhaps rename the top-level values to `[manifest]ListBytes`/`[manifest]ListType` to make it very clear that the two have nothing in common.

Reply:
Renamed the top-level values.  Moved initialization of `lists` up one scope since we were doing it in both branches of an `if/then`, and moved initialization of `list` into the only scope where it's used.